store/actions.ts

```diff
@@ -5,8 +5,21 @@ import { adjustMultistoreApiUrl } from '@vue-storefront/core/lib/multistore'
 
 // it's a good practice for all actions to return Promises with effect of their execution
 export const actions: ActionTree<PaypalState, any> = {
-  complete ({}, params) {
-    let url = config.paypal.endpoint.complete
+  complete({ }, params) {
+    let url = config.paymentPaypalMagento2.endpoint.complete
```

Comment:
@tonyisworking All action should return a `Promise`, for use `async/await` in methods.
+ @collymore